Source/tess.c

```diff
@@ -89,6 +89,11 @@ static void ComputeNormal( TESStesselator *tess, TESSreal norm[3] )
 	int i;
 
 	v = vHead->next;
+	if (v == vHead) {
+		/* No vertex is initialized -- normal doesn't matter */
```

Comment:
Thanks for the comment! I think it is really important to leave comments when fixing bugs, since the solution usually was not obvious in the first place.